resources/views/tools/database/edit-add-bread.blade.php

```diff
@@ -1,3 +1,12 @@
+<?php
+    $types = [
+        'number' => ['smallint', 'mediumint', 'integer', 'float', 'double', 'decimal'],
+        'date' => ['datetime', 'date'],
+        'textarea' => ['bigtext', 'text', 'mediumtext', 'longtext'],
```

Comment:
This would be better stored in the form field object.
Example for the `textarea` it could be stored here: https://github.com/the-control-group/voyager/blob/master/src/FormFields/TextAreaHandler.php